src/styles/index.scss

```diff
@@ -1,3 +1,13 @@
+@import 'main';
+
 body {
   margin: 0;
+  padding: 0;
+  width: 100%;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-left: 5px;
```

Comment:
The `body` tag should not have a margin-left property as it causes the whole page to be misaligned. Consider removing the `margin-left: 5px;` on line 12 to adhere to the standard practice of keeping the body element's margin consistent on all sides.